src/main/liquibase/liquibase-ciel-data-18x.xml

```diff
@@ -21,8 +21,8 @@
 	</changeSet>
 	
 	<changeSet id="20120315-1100" author="standalone" dbms="mysql">
-		<comment>Adding mvp data</comment>
-		<sqlFile path="openmrs_concepts_${ciel.dictionary.openmrs.version}_${ciel.dictionary.version}.sql" encoding="UTF-8" />
+		<comment>Adding ciel data</comment>
+		<sqlFile path="openmrs_concepts_${ciel.dictionary.openmrs.version}_${ciel.dictionary.version}.sql" encoding="UTF-8" ></sqlFile>
```

Comment:
Please change this line back so it reads `<sqlFile path="openmrs_concepts_${ciel.dictionary.openmrs.version}_${ciel.dictionary.version}.sql" encoding="UTF-8" />`